.github/workflows/release-binaries.yml

```diff
@@ -32,6 +32,15 @@ jobs:
           asset_name: darwin-x64
           asset_content_type: application/octet-stream
 
+      - uses: actions/upload-release-asset@v1.0.1
```

Comment:
[optional] Having even more repetitive lines here now, we *could* create a custom action that takes `tag_name`, `release_name`, `github_token` & a list of assets and calls `actions/create-release@v1` & `actions/upload-release-asset@v1.0.1` underneath.

Reply:
I didn't have a way to test this. I'm not familiar enough with Actions to know why it is not finding the actions in my fork. When I open the Actions tab in GitHub I cannot run this workflow, so I was only able to make the safe copy-paste changes.

Reply:
Alright, fair enough.